cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -96,10 +96,11 @@ def test_order_catalog_item(catalog_item, request):
     catalog_item.create()
     service_catalogs = ServiceCatalogs("service_name")
     service_catalogs.order(catalog_item.catalog, catalog_item)
-    logger.info('Waiting for cfme provision request for service %s', catalog_item.name)
+    logger.info('Waiting for cfme provision request for service {}'.format(catalog_item.name))
     row_description = catalog_item.name
     cells = {'Description': row_description}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
         fail_func=requests.reload, num_sec=1400, delay=20)
-    assert row.last_message.text == 'Service Provisioned Successfully'
+    assert row.last_message.text == '[EVM] Service [{}] Provisioned Successfully'.\
```

Comment:
We also potentially want to "variabilize" the `EVM` since that can change (server name)